docs/docs-beta/docs/dagster-plus/insights.md

```diff
@@ -1,5 +1,157 @@
 ---
 title: "Insights"
+description: "Visiblity into historical usage, cost, and metadata."
 ---
 
-# Dagster+ Insights
\ No newline at end of file
+# Dagster+ Insights
+
+Using Dagster+ Insights, you can gain visibility into historical usage and cost metrics such as Dagster+ run duration, credit usage, and failures.
+Visualizations are built right into the Dagster UI, allowing you to explore metrics from Dagster as well as downstream systems (like Google BigQuery) all in one place.
+
+### With Insights, you can
+
+- [Explore usage trends in your Dagster pipelines](#explore-dagsters-built-in-metrics)
+- [Integrate additional metrics](#integrate-other-metrics), like data warehouse cost or your own custom metadata
+- [Export metrics](#export-metrics) from Dagster+
+- [Create alerts](/dagster-plus/deployment/alerts#alerting-when-an-insights-metric-crosses-a-threshold) based off of Insights metrics {/* TODO: write this alerts section) */}
+
+<details>
+  <summary>Prerequisites</summary>
+
+To use Insights, you'll need a Dagster+ account.
+</details>
+
+## Explore Dagster's built-in metrics
+
+To access Insights, click **Insights** in the top navigation bar in the UI:
+
+![Viewing the Insights tab in the Dagster+ UI](/img/placeholder.svg)
+{/* <Image
+alt="Viewing the Insights tab in the Dagster UI"
+src="/images/dagster-cloud/insights/insights-tab.png"
+width={2640}
+height={1516}
+*/}
+
+The left navigation panel on this page contains a list of available metrics. For each metric, the daily, weekly, or monthly aggregated values are displayed in the graph.
+
+Use the tabs above the charts to view metrics for **Assets**, **Asset groups**, **Jobs**, and **Deployments**.
+
+These metrics are updated on a daily basis. Refer to the [built-in metrics](#built-in-metrics) section for more information about what Dagster provides out of the box.
+
+
+## Working with Insights metrics
+
+### Data retention
+
+How long historical Insights data is retained depends on your Dagster+ plan:
+
+- **Dagster+ Pro** - 120 days
+- **All other plans** - 30 days
+
+### Built-in metrics
```

Comment:
This and the data retention section are definitely more "reference" than "guide", but I wasn't sure where else to include them.

Reply:
Yeah, I agree. But let's leave it for now and we'll figure out if there's a better home for it later.